src/feedcontainer.cpp

```diff
@@ -109,6 +109,45 @@ void FeedContainer::sort_feeds(const FeedSortStrategy& sort_strategy)
 			}
 		});
 		break;
+	case FeedSortMethod::LATEST_UNREAD:
+		std::stable_sort(
+			feeds.begin(), feeds.end(),
+		[&](std::shared_ptr<RssFeed> a, std::shared_ptr<RssFeed> b) {
+			if (a->unread_item_count() == 0 || b->unread_item_count() == 0) {
```

Comment:
If I understand correctly, the current implementation sorts feeds by their most recent unread articles.
For feeds which have no unread articles, the order is kept unchanged.
(except in case of reverse sort, where it seems the order is flipped every time it is selected)

I'm wondering if it makes sense to sort the "read" feeds as well based on their most recent article.
I've experimented with that for a bit locally, I might share that in a bit.

Reply:
Yes, I think also sorting feeds with no unread on latest "read" can make sense.

Reply:
I haven't really found time to test my suggested changes, so I will instead merge this as-is (and maybe apply the improvement at a later time)